.changeset/spotty-bananas-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/firestore': major
+---
+
+Fixed a bug where a rejected promise with an empty message in a transaction would cause a timeout. (https://github.com/firebase/firebase-js-sdk/issues/9147)
```

Comment:
I believe you can remove this link to the issue, but you need to update your PR description to use this exact syntax:

```
Fixes: #9147
```

Then the change log generator will be able to parse this issue number out of the PR description and generate the link for you.

Reply:
Also, this would be a "patch", not a "major" bump, right? See line 2.

Reply:
Please also remove the http link and add the line "Fixes: #9147" to the PR description. Then the release notes will be automatically populated with the link to the issue. For an example of this, see https://github.com/firebase/firebase-js-sdk/pull/9151

Reply:
Thank you! Sorry I didn't correct it in the first time. Please let me know if anything else need to be changed.